src/Scaffolder/Declaration/AbstractEntityDeclaration.php

```diff
@@ -78,7 +79,24 @@ protected function isNullableType(string $type): bool
 
     private function variableType(string $type): string
     {
-        return $this->isNullableType($type) ? (substr($type, 1) . '|null') : $type;
+        if ($this->isNullableType($type)) {
+            $type = \substr($type, 1);
+        }
+
+        $phpMapping = [
+            'int'   => ['primary', 'bigPrimary', 'integer', 'tinyInteger', 'smallInteger', 'bigInteger'],
+            'bool'  => ['boolean'],
```

Comment:
```suggestion
            'bool'  => ['boolean', 'bool'],
```
Is `bool` valid value?

Reply:
Valid, thanks, added aliases:
https://github.com/cycle/database/blob/2.x/src/Schema/AbstractColumn.php#L210